src/utils.ts

```diff
@@ -505,6 +527,155 @@ export function isNodeModule(path: string) {
   return /^([A-z0-9@])/.test(path)
 }
 
+type NexusWrapKind = { type: 'List'; nullable: boolean } | { type: 'WrappedType'; nullable: boolean }
+
+export function unwrapNexusDef(
+  typeDef: AllNexusTypeDefs | AllNexusArgsDefs | string,
+  nonNullDefault: boolean
+): { namedType: AllNexusNamedTypeDefs | AllNexusArgsDefs | string; wrapping: NexusWrapKind[] } {
+  const wrapping: NexusWrapKind[] = []
+  let namedType = typeDef
+
+  const isWrappedType = (obj: any): obj is AllNexusNamedTypeDefs | NexusArgDef<any> | string =>
```

Comment:
```suggestion
  const isNamedType = (obj: any): obj is AllNexusNamedTypeDefs | NexusArgDef<any> | string =>
```

Reply:
Or negate meaning here and then down below do `!isWrappedType(...)` which seems easier to understand.

Reply:
done!

Reply:
The reason it was called `WrappedType` is because there can be NexusArgDef in here, which technically aren't named types. But it doesn't matter